astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -591,25 +593,26 @@ def _draw_grid(self, renderer):
 
         renderer.close_group('grid lines')
 
-    def _draw_ticks(self, renderer, bboxes, ticklabels_bbox, ticks_locs):
+    def _draw_ticks(self, renderer, ticklabels_bbox, ticks_locs):
 
         renderer.open_group('ticks')
 
         self.ticks.draw(renderer, ticks_locs)
-        self.ticklabels.draw(renderer, bboxes=bboxes,
+
+        self.ticklabels.draw(renderer, bboxes=None,
                              ticklabels_bbox=ticklabels_bbox,
                              tick_out_size=self.ticks.out_size)
 
         renderer.close_group('ticks')
 
-    def _draw_axislabels(self, renderer, bboxes, ticklabels_bbox, ticks_locs, visible_ticks):
+    def _draw_axislabels(self, renderer, ticklabels_bbox, ticks_locs, visible_ticks):
         # Render the default axis label if no axis label is set.
         if self._auto_axislabel and not self.get_axislabel():
             self.set_axislabel(self._get_default_axislabel())
 
         renderer.open_group('axis labels')
 
-        self.axislabels.draw(renderer, bboxes=bboxes,
+        self.axislabels.draw(renderer, bboxes=None,
```

Comment:
Ideally one would remove the `bboxes` parameter, but this would break the signature. Not sure how best to handle this, ideally with a deprecation warning?